indexbuilder.go

```diff
@@ -368,12 +368,11 @@ func (b *IndexBuilder) symbolKindID(t string) uint32 {
 func (b *IndexBuilder) addSymbols(symbols []*Symbol) {
 	for _, sym := range symbols {
 		b.symMetaData = append(b.symMetaData,
-			b.symbolID(sym.Sym),
+			0,
```

Comment:
why are we still storing this? For backwards compatibility on v16?

Reply:
Yes, I hope to remove the first uint32 entirely in the future. This would most likely require a change in the feature version, thus reindexing. For now, setting it to `0` will remove the redundant storage of the symbol names.

Reply:
Cool, maybe add a TODO explaining that. IE we can remove it, it will just require bumping feature version.